web/impact/impact/models/utils.py

```diff
@@ -28,3 +27,11 @@ def model_name_to_snake(value):
 
 def snake_to_model_name(value):
     return "".join(map(str.title, value.split("_")))
+
+
+def is_int(s):
```

Comment:
A little bit of bikeshedding - I like this function but the name is potentially misleading. 

    >>> is_int(3.14159)
    True

Something like `can_cast_to_int` would be more precise. Not a blocker but it would make me happy.

Reply:
Actually I'd expect `is_int(3.14159)` to be `False`. fixing.